tests/integration/s2n_handshake_test_gnutls.py

```diff
@@ -36,8 +36,8 @@ def try_gnutls_handshake(endpoint, port, priority_str):
     s2nd.stdout.readline()
 
     # Fire up gnutls-cli, use insecure since s2nd is using a dummy cert
-    gnutls_cli = subprocess.Popen(["gnutls-cli", "--priority=" + priority_str,"--insecure", "-p " + str(port), str(endpoint)], 
-                                   stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.DEVNULL)
+    gnutls_cli = subprocess.Popen(["gnutls-cli", "--priority=" + priority_str,"--insecure", "--recordsize=" + "512",
```

Comment:
Attaching the screenshot of wireshark showing successful TLS handshake (GnuTLS client & S2N server) with MFL extension of 512.

![gnutls_s2n_512](https://user-images.githubusercontent.com/5386340/29052001-a9a99710-7b9b-11e7-91c0-b116c5837728.png)